### PR TITLE
Rainbow thrower spawner and unequip lifetime

### PR DESCRIPTION
**Pull request by RebeccaStankus**
TEST PLAN
-Put the "rainbowThrowerSpawner" script on an entity.
-A new invisible rainbow thrower should appear. The spawner entity checks every 5 seconds to see if it needs to create a new rainbow thrower.
-Equip the rainbow thrower. It should become visible and have a lifetime of -1.
-Unequip the rainbow thrower. It should remain visible and have a lifetime that is not -1. It will disappear in 30 seconds.
-Equip another rainbow thrower. Unequip and equip again. The lifetime should be -1 again while equipped.

### CODE REVIEW

File: DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js [hunk 1]
```diff
@@ -1,643 +1,689 @@
...
+                showOverlay = false;    
+            }
+            
+            if (showOverlay) {
+                var viewport = Controller.getViewportDimensions();
+                var windowHeight = viewport.y;
+                desktopHowToOverlay = Overlays.addOverlay("image", {
+                    imageURL: DESKTOP_HOW_TO_IMAGE_URL,
+                    x: 0,
+                    y: windowHeight - DESKTOP_HOW_TO_IMAGE_HEIGHT - 24,
```
Comment (elisa-lj11) [inline DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js]: does 24 need to be hard-coded?

File: DomainContent/Hub/RainbowThrower/scripts/rainbowThrowerGunSpawner.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        
+        unload: function() {
+            if (checking) {
+                Script.clearInterval(checking);
+            }
+        }
+    };
+
+    return new RainbowThrowerSpawner();
+});
```
Comment (elisa-lj11) [inline DomainContent/Hub/RainbowThrower/scripts/rainbowThrowerGunSpawner.js]: newline

File: DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js (hunk 1 above)
Comment (RebeccaStankus) [reply DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js]: checking with David...

Comment (RebeccaStankus) [reply DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js]: fixed with last commit

Comment (elisa-lj11) [reply DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js]: I don't see the fix in the most recent push

**Pull request merged**